source/sass/views/youtube-regrets-2022.scss

```diff
@@ -0,0 +1,57 @@
+.youtube-regrets-2022 {
+  $font-zilla: "Zilla Slab", sans-serif;
+
+  .recommendations {
+    background-color: $black;
+    padding-top: 85px;
+
+    &__title {
+      font-size: 30px;
+      line-height: calc(34 / 30);
+
+      @media screen and (min-width: $bp-md) {
+        font-size: 52px;
+        line-height: calc(60 / 52);
```

Comment:
This equals 1.15 too so we don't need it here haha

```suggestion
```

Reply:
If we do this, can we use values from the design system?